backend/tests/modules/simulation/test_persist_progress.py

```
@@ -0,0 +1,470 @@
+            current_scene_id=100,
+            orchestrator_data=sample_orchestrator_data,
+            simulation_status="in_progress",
+            session_count=1,
+            last_activity=datetime.utcnow() - timedelta(hours=1),
+            scenes_completed=[],
+        )
+        db_session.add(up)
+        db_session.commit()
+        db_session.refresh(up)
+        return up
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)